src/script/user/UserRepository.ts

```diff
@@ -425,14 +404,13 @@ export class UserRepository {
       messageId: createRandomUuid(),
     });
 
-    const recipients = this.isTeamTooLargeForBroadcast() ? [this.self()] : teamUsers.concat(this.self());
-    amplify.publish(WebAppEvents.BROADCAST.SEND_MESSAGE, {genericMessage, recipients});
+    const sortedUsers = this.directlyConnectedUsers().sort(({id: idA}, {id: idB}) =>
+      idA.localeCompare(idB, undefined, {sensitivity: 'base'}),
+    );
+    const [members, other] = partition(sortedUsers, user => user.isTeamMember());
+    const recipients = [this.self(), ...members, ...other].slice(0, UserRepository.CONFIG.MAXIMUM_TEAM_SIZE_BROADCAST);
 
```

Comment:
The recipients of the status update are now determined as: [self, ...team members, ...other connected users]. This list is  cut after 500. To make sure that every platform would send to the same 500 users, they are sorted by their user id.